airbyte-connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -144,6 +144,9 @@ components:
           type: object
           description: The AirbyteStateMessage object to use as the starting state for this read
           # $ref: "#/components/schemas/AirbyteProtocol/definitions/AirbyteStateMessage"
+        record_limit:
+          type: integer
+          description: Number of records that will be returned to the client from the connector builder (max of 1000)
```

Comment:
the minimum and maximum could be defined here so the FE doesn't have to redefine it https://swagger.io/docs/specification/data-models/data-types/#numbers

Reply:
Done.